test-files/cases/hoot-js/line-generic-1/Config.conf

```diff
@@ -1,8 +1,10 @@
 {
-  "match.creators" : "hoot::ScriptMatchCreator,LineStringGenericTest.js",
+  "match.creators" : "hoot::ScriptMatchCreator,LineStringGeneric.js",
   "merger.creators" : "hoot::ScriptMergerCreator",
   "uuid.helper.repeatable" : "true",
   "generic.line.match.threshold" : "0.6",
   "generic.line.miss.threshold" : "0.6",
-  "generic.line.review.threshold" : "0.6"
+  "generic.line.review.threshold" : "0.6",
+  "search.radius.generic.line" : "15.0",
+  "#" : "end"
```

Comment:
Using the `CornerSplitter` with only the `angleHistogramExtractor` in `LineStringGenericTest.js` results in bad conflation because of the smaller line segments that cause the angle histogram to go outside of `0.63` instead of being smoothed over a larger distance.  The full checks in `LineStringGeneric.js` allow the toy test to conflate successfully.